webpack.config.js

```diff
@@ -11,7 +12,7 @@ Encore
     .setOutputPath('public/build/')
 
     // public path used by the web server to access the output path
-    .setPublicPath(Encore.isProduction() ? '/pelagos-symfony/build' : '/build')
+    .setPublicPath(arguments.publicpath ? arguments.publicpath : '/build')
```

Comment:
Maybe we should add a "dev" environment too, just to keep the usage clean. IDK.

Reply:
Default is dev, so always "/build". `yarn dev` will use default path.